src/sagas/import-project.saga.js

```diff
@@ -0,0 +1,160 @@
+import { call, put, cancel, select, takeEvery } from 'redux-saga/effects';
```

Comment:
Would it be possible to add Flow typing to this file?

I'm not as worried about `dependency.saga.js` for two reasons:

- The original `dependency.middleware.js` didn't have Flow typing, which was my bad
- @superhawk610's #103 PR adds Flow typing for this file.

If you're not comfortable/familiar with Flow, or if you just don't have the time to spare right now, feel free to just add a `// TODO: Add Flow` to the top of the file, and I or someone else will try to tackle it shortly after merging.

Reply:
Sure, added